config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: controller
+  newName: ghcr.io/aenix-io/etcd-operator
```

Comment:
@sircthulhu the reason, why kustomization does not work for image is in "name" field: we need either change it to `ghcr.io/aenix-io/etcd-operator` (as it is in manager.yaml) in kustomization.yaml or change the image name to `controller` in deployment manifest (manager.yaml)